algorithms/C/searching/linearsearch.c

```diff
@@ -0,0 +1,67 @@
+// linear search
+#include <stdio.h>
+#include <stdlib.h>
+//function prototype
+int linearSearch(int a[], int query, int size);
+
+int main()
+{
+   int *data,i,temp,searchQuery,size;
+   scanf("%d",&size);
+   data=(int*)malloc(sizeof(int)*size);
+
+   //generate some data
+   for (i = 0; i < size; ++i)
+   {
+      scanf("%d",&data[i]);
+   }
+
+   printf("Enter integer to search for: ");
+   scanf("%d", &searchQuery);
+
+   //finding searchQuery in array data
+   temp = linearSearch(data, searchQuery, size);
+
+   //condition for displaying result
+   if ( temp != -1)
+      printf("Found value and the index is :%d", temp);
```

Comment:
```suggestion
      printf("Value found at index no: %d", temp);
```
your sentence has not perfect sense